superset-frontend/src/chart/chartReducer.ts

```diff
@@ -134,7 +143,7 @@ export default function chartReducer(charts = {}, action) {
       }
       const annotationQuery = {
         ...state.annotationQuery,
-        [action.annotation.name]: action.queryRequest,
+        [action.annotation.name]: action.queryController,
```

Comment:
Bycatch by TypeScript: it seems the `ANNOTATION_QUERY_STARTED` is supposed to cancel previous annotation queries, but it used the wrong action prop. The source of the action is here:

https://github.com/apache/superset/blob/6731cd0be6852c890a5e856e79470d08eb760437/superset-frontend/src/chart/chartAction.js#L97